lessons/python/intro-nlp/intro_NLP.ipynb

```diff
@@ -0,0 +1,3910 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7f049e52c6d8>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fdist.plot(30)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "# Stopwords"
```

Comment:
s/Stopwords/Stop words/

Some explanation of what stop words are would be useful.